NineChronicles.Standalone.Executable/Program.cs

```diff
@@ -55,9 +56,15 @@ static void ConfigureSentryOptions(SentryOptions o)
             string host = null,
             [Option('P')]
             ushort? port = null,
+            [Option("swarm-private-key",
+                Description = "The private key used for signing messages and to specify your node. " +
+                              "If you leave this this null, randomly generated value will be used.")]
```

Comment:
As `Block.Miner` field can be null, it doesn't have to randomly generate a key.

Reply:
`swarm-private-key` is used for peer-to-peer data transferring. `private-key` is used in mining and it is not randomly generated. Do you mean it should not throw error and make it be null?

Reply:
Ah, fair enough.